liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojoSupport.java

```diff
@@ -291,9 +291,17 @@ private void addLibrary(Element parent, LooseApplication looseApp, String dir, A
                 String artifactFileName = getPreDeployAppFileName(dependProject);
                 Element archive = looseApp.addArchive(parent, dir + artifactFileName);
                 looseApp.addOutputDir(archive, new File(dependProject.getBuild().getOutputDirectory()), "/");
-                
-                File manifestFile = MavenProjectUtil.getManifestFile(dependProject, "maven-jar-plugin");
-                looseApp.addManifestFileWithParent(archive, manifestFile);
+
+                log.info(dependProject.getPackaging());
```

Comment:
Did you mean to leave this in? If so, probably should add a little more info to it.

Reply:
Nope. I'll remove it.